src/main/java/Car.java

```diff
@@ -0,0 +1,42 @@
+package RacingCar;
```

Comment:
패키지 이름 컨벤션은 소문자입니다.
아래에도 보면 
`import java.util.List`가 있습니다. `List` 클래스는 `java` 패키지 아래 `util` 패키지 아래에 있습니다.
```suggestion
package racingcar;
```